src/components/tx/TxData.js

```diff
@@ -551,10 +557,20 @@ class TxData extends React.Component {
     }
 
     const renderConfirmationLevel = () => {
+      function getConfirmationMessage(data) {
+        if (!data) {
+          return 'Retrieving confirmation level data...';
+        }
+        if (!data.success) {
+          return 'Not available';
+        }
+
+        return `${helpers.roundFloat(data.confirmation_level * 100)}%`
+      }
```

Comment:
Instead of showing `NaN%` show a friendly error message when we have errors on the api.